app/views/problem_reports/new.html.erb

```diff
@@ -1,48 +1,51 @@
-<div class="container-fluid">
-  <h3> Create New Problem Report </h3>
+<div class="container-fluid no-padding">
+  <h1> Create New Problem Report </h1>
```

Comment:
I think for this Herbert also would've asked to remove spaces haha

Reply:
And this too! I don't see the spaces? How do you guys see the spaces. ):